src/popup/router/components/InputField.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="input-field">
+    <label class="label" v-if="label">
+      {{ label }}
+    </label>
+    <div :class="{ focus, error }" class="container">
+      <slot v-if="!error" name="left" />
+      <Error v-else />
+      <input
+        :type="type"
+        class="input"
+        :placeholder="placeholder"
+        :value="value"
+        @input="$emit('input', $event.target.value)"
+        @focus="(f) => (focus = f)"
```

Comment:
```suggestion
        @focus="focus = true"
        @blur="focus = false"
```
don't you forget to reset the focus state on blur? Also, proposing to rename `focus` to `focused` or `isFocused`